docs/quickstart.md

```diff
@@ -166,6 +166,21 @@ build/generated/ksp/main/java/
 ```
 
 and perhaps also resource directory if your IDE supports them:
+
+```
+build/generated/ksp/main/resources/
 ```
-build/generated/ksp/main/resources
+
+It may also be necessary to configure these directories in your KSP consumer module:
```

Comment:
May I know in what case that marking those dirs in IDE doesn't work? E.g., some IDE may not support marking generated sources?

Reply:
I have this problem with IntelliJ in all my projects using KSP. Marking the directory with generated sources makes code highlighting / symbol resolution work when opening the generated files, however application code referencing generated files still present an error:

![bark-ide-issue-1](https://user-images.githubusercontent.com/1481940/129059440-e35e3616-8057-47b1-b3d3-0d9230fe8320.png)

Adding the directories to the source sets solves it.

I thought this was a common issue, since a few people commented in #37.

Reply:
I see. Let's merge this and allow me to investigate on why the IDE marking doesn't work later. Thanks for the PR!

Reply:
Cool, thank you!